mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -2,6 +2,22 @@
 
 # Deploy the management hub services (agbot, exchange, css, sdo, postgre, mongo), the agent, and the CLI on the current host.
 
+# Get current hardware architecture
+export ARCH=$(uname -m | sed -e 's/aarch64.*/arm64/' -e 's/x86_64.*/amd64/' -e 's/armv.*/arm/')
+if [ "${ARCH}" = "ppc64le" ]; then
+    export ARCH_DEB=ppc64el
+else
+    export ARCH_DEB="${ARCH}"
+fi
+
+# Set the correct default value for docker-compose command
+if [ -f /usr/bin/docker-compose ]; then
+    export DOCKER_COMPOSE_CMD="docker-compose"
+else
+    export DOCKER_COMPOSE_CMD="pipenv run docker-compose"
```

Comment:
There is a special requirement to the minimal version of Docker Compose in the `deploy-mgmt-hub.sh` script: 1.21.0. So I had to use Python-based version of it because in the standard Ubuntu and RHEL repos on pcc64le the latest version is 1.18 only and the official Github repo doesn't support ppc64le binaries. If I don't specify version during Python-based Docker Compose installation trying to get the latest one the installation hangs up by some unclear reasons in Ubuntu and RHEL. So I used minimal required version to fix this issue in both OSs also there are more newer ones in repo (1.24.0 if I remember correctly but not sure the available version is the same in both OSs).